homeassistant/components/generic/config_flow.py

```diff
@@ -91,6 +91,12 @@ def build_schema(
             CONF_PASSWORD,
             description={"suggested_value": user_input.get(CONF_PASSWORD, "")},
         ): str,
+        vol.Optional(
+            CONF_CONTENT_TYPE,
+            description={
+                "suggested_value": user_input.get(CONF_CONTENT_TYPE, "image/jpeg")
+            },
+        ): str,
```

Comment:
Why are we asking the user for input in that case? Do we know when only a stream is given already?

Reply:
We definitely can move it in a second step when we know the still image URL is empty.